custom/enikshay/case_utils.py

```diff
@@ -249,6 +271,16 @@ def get_open_referral_case_from_person(domain, person_case_id):
         )
 
 
+def get_latest_closed_referral_case_from_person(domain, person_case_id, with_status=None):
+    closed_referral_cases = get_referral_cases_from_person(domain, person_case_id, closed_cases=True)
+    if with_status:
+        closed_referral_cases = filter(
```

Comment:
this is fine, but curious why you used a `filter` here instead of a list comprehension like we've been using elsewhere in this file.

Reply:
no preference. Will update it to keep it similar to what is being used.

also checked that `filter` is probably not favoured. thanks for flagging.